bundles/org.openhab.binding.modbus.solaxx3mic/README.md

```diff
@@ -0,0 +1,296 @@
+# modbus.SolaxX3Mic Binding
+
+This binding has been developed to handle Modbus communication via RS-458 port of Solax X3 Mic PhotoVoltaic Inverter.
+Handles collection of all important information about inverters state.
+
+It has been found that every inverter has own map of registers and really small amount of available inverters are following SunSpec standard.
```

Comment:
Definitely not. SunSpec is much wider with support for several different profiles.
I just got two interters and found that they have almost everything in common, but presented in different modbus registers.
It can be said that SunSpec is superset of this binding.